### DISCUSSION TITLE
BASS

### DISCUSSION THREAD

**Discussion by QuartzWasTaken** [ppy/osu#22559] [Q&A]
### Type

Other

### Bug description

When switching the audio output from my PC (which doesn't have speakers) to my mic's (blue yeti), the game did switch the output to my mic, but gave me the error : BASS faulted with error code -1: Unknown (as seen in screenshot)
This error isn't too annoying, because it still switched the output, I'd just like to see it fixed :)
Edit : sorry for the issue title, I thought I changed it to something easier to figure out before posting it

### Screenshots or videos

![image](https://user-images.githubusercontent.com/68695198/217567972-1cd4d2bc-dca8-43fa-bd49-aff15863bb3f.png)

### Version

2023.207.0

### Logs

[updater.log](https://github.com/ppy/osu/files/10686918/updater.log)
[database.log](https://github.com/ppy/osu/files/10686919/database.log)
[input.log](https://github.com/ppy/osu/files/10686920/input.log)
[network.log](https://github.com/ppy/osu/files/10686921/network.log)
[performance.log](https://github.com/ppy/osu/files/10686922/performance.log)

**Comment by zoey-on-github** [comment]
does this error only happen when going from PC to mic, or does it also still happen when you switch output back from your mic to your PC?

**Comment by bdach** [comment]
Can you attach `runtime.log` when this happens? Your opening post is missing it, and it's the one most likely to be relevant in this case.
